package.json

```diff
@@ -90,9 +90,9 @@
     "postcss-cli": "4.1.0",
     "prettier": "^1.4.4",
     "prop-types": "15.5.8",
-    "react": "15.5.4",
-    "react-dom": "15.5.4",
-    "react-test-renderer": "15.6.1",
+    "react": "15.5.4 || ^16.0.0",
+    "react-dom": "15.5.4 || ^16.0.0",
+    "react-test-renderer": "15.5.4 || ^16.0.0",
```

Comment:
I'd probably make the dev deps more explicit. Also, I think you may have changed the version here by accident.